gfx/MShockXotto+/pngs_tiles_32x32/character/overlay/wielded/hammers/overlay_wielded_hammer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "overlay_wielded_hammer",
+  "fg": [
+    "overlay_wielded_hammer"
```

Comment:
The json is not necessary if id and fg are the same thing

Reply:
I do need this here; for some reason the composer isn't adding it to the compiled json - since I ask the github machines to compile the tileset for me I can link you this, so you can check it out for yourself

this workflow https://github.com/casswedson/CDDA-Tilesets/actions/runs/2444709932 runs off this commit https://github.com/casswedson/CDDA-Tilesets/commit/e4da325dd8d1e32ddc788e7da63fb52d1e2bd4fd#diff-f915b6a6e38472bd377a9a86de0a00719202a1b7a7cc6399f675e2ad43b90d27 and does not contain an entry for overlay_wielded_hammer despite being a sprite by that name

![Screenshot from 2022-06-06 17-04-15](https://user-images.githubusercontent.com/58050969/172257119-3220fba7-bdfc-4eaf-ac9d-e355534b2108.png)

here is a screenshot if you don't want to all the steps